dashboard/app/dsl/base_dsl.rb

```diff
@@ -20,8 +20,9 @@ def self.parse_file(filename, name=nil)
   def self.parse(str, filename, name=nil)
     object = new
     object.name(name) if name.present?
-    ascii = str ? str.to_ascii : ''
-    object.instance_eval(ascii, filename)
+    # Use consistent apostrophe symbol, and remove no-break space
+    parsed_string = str ? str.gsub(/‘|’/, "'").gsub(/ /, "") : ''
```

Comment:
Not sure if it's possible to do two different substitute-and-replace on the same pass of the string, so I'm doing two separate ones.

It's hard to see on the GitHub editor, but the apostrophes replaced are the "right single quotation mark" (U+2019) and the "left single quotation mark" (U+2018). The "grave accent" (U+0060) is what's used to demarcate code (i.e. `used here`) and is *not* being replaced. Used this site https://unicode-table.com/en/ for reference.

`gsub(/ /, "")` looks silly, but it's removing the no-break space (on RubyMine, the character shows as the following:
![image](https://user-images.githubusercontent.com/9142121/153220680-511aea7e-c255-4034-b169-18dffba61648.png)
The original `to_ascii` method also strips this character, so the functionality should stay the same.

Reply:
Ah, good point (and good call testing it there!) Updated it to fix it.